flexmeasures/data/utils.py

```diff
@@ -44,3 +49,112 @@ def get_data_source(
             f'Session updated with new {data_source_type} data source "{data_source.__repr__()}".'
         )
     return data_source
+
+
+def save_to_db(
```

Comment:
The concept of committing is very important and needs to be more explicit. I suggest we either

- put the word "commit" in the function name
- add a `commit: bool = False` parameter, so when using this function you need to show you want that.